userspace/libscap/scap.c

```diff
@@ -1231,8 +1231,17 @@ int scap_get_events_from_ppm_sc(IN uint32_t ppm_sc_array[PPM_SC_MAX], OUT uint32
 			struct syscall_evt_pair pair = g_syscall_table[syscall_nr];
 			if(pair.ppm_sc == ppm_code)
 			{
-				events_array[pair.enter_event_type] = 1;
-				events_array[pair.exit_event_type] = 1;
+				int enter_evt = pair.enter_event_type;
```

Comment:
This is a fix: for syscall table entries that only have a `.ppm_sc` code set, we should consider `PPME_GENERIC_X` as exit event (instead of zero-initialized `PPME_GENERIC_E`).